.github/workflows/robustness-nightly.yaml

```diff
@@ -14,6 +15,14 @@ jobs:
       count: 100
       testTimeout: 200m
       artifactName: main
+  main-arm64:
+    uses: ./.github/workflows/robustness-template.yaml
+    with:
+      etcdBranch: main
+      count: 100
+      testTimeout: 200m
+      artifactName: main-arm64
+      runs-on: "['self-hosted', 'Linux', 'ARM64']"
```

Comment:
```suggestion
      runs-on: [self-hosted, Linux, ARM64]
```

Reply:
I got the impression from https://github.com/orgs/community/discussions/11692#discussioncomment-3541856 that the single quotes were required for the hack to work with arrays to function?

Reply:
Feel free to keep it as it's if it doesn't break anything. 

I raised this comment because the github official examples do not have quotes (neither double quote, nor single quote)